main.go

```diff
@@ -164,6 +165,24 @@ func (c *runCmd) Run() error {
 	return nil
 }
 
+func (c *runCmd) fileBytes() ([]byte, error) {
+	b, err := fileBytes(c.Source)
+	if err != nil {
+		return nil, err
+	}
+	if filepath.Ext(c.Source) == ".txtar" && c.Txtar != "" {
```

Comment:
Perhaps add another case where `c.Txtar != ""` and the source is not a .txtar file and return an error?